_zephyr.pyx

```diff
@@ -187,24 +187,6 @@ def sub(cls, instance, recipient):
     errno = ZSubscribeTo(newsub, 1, 0)
     __error(errno)
 
-def subAll(lst):
-    cdef ZSubscription_t *newsubs
-    cdef unsigned int i
-
-    newsubs = <ZSubscription_t*>calloc(len(lst), sizeof(ZSubscription_t))
-    try:
-        for 0 <= i < len(lst):
-            newsubs[i].zsub_class = lst[i][0]
-            newsubs[i].zsub_classinst = lst[i][1]
-            newsubs[i].zsub_recipient = lst[i][2]
-
-        errno = ZSubscribeTo(newsubs, len(lst), 0)
-        __error(errno)
-    finally:
-        if newsubs:
-            free(newsubs);
-    pass
```

Comment:
AFAICT subAll isn't used in PyZephyr itself anywhere, and based on the name I'm assuming that nothing in _zephyr is intended to be "public" (except for receive and ZNotice, of course, which are imported into zephyr's namespace). subAll didn't build (I think due to the byte/string mismatch), so I just removed it rather than trying to fix it. I could fix it instead.